src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -102,6 +102,16 @@ def get_instance_url() -> str:
     return "https://%s.azurewebsites.net" % get_instance_name()
 
 
+@cached
+def get_instance_id() -> UUID:
+    from .containers import get_blob
+
+    blob = get_blob("base-config", "instance-id", account_id=get_func_storage())
+    if blob is None:
+        raise Exception("missing instance_id")
```

Comment:
Should this be `instance-id` in the exception message? Thinking about greppability.

Reply:
Oh, actually, is the `instance-id` string above just incorrect? Looks like we set the snake case version below.